### DISCUSSION TITLE
Is there a way to be able to interact with the background components when Modal is visible?

### DISCUSSION THREAD

**Discussion by sankomil** [react-native-modal/react-native-modal#601] [Q&A]
Hi, as the title states, I am attempting to implement a modal in such way that it only takes up some part of the screen so as to allow the user to be able to interact with some of the background components. The current screen looks like this:

![image](https://user-images.githubusercontent.com/43489998/132301087-381bcf23-5375-47ca-a160-c5a82c37ca24.png)

My current modal looks like this:

```tsx
<Modal
  isVisible={showModal}
  style={styles.modalContainer}
  customBackdrop={<CustomBackdrop />}
  onBackdropPress={() => setShowModal(false)}
  animationIn="fadeInDown"
  animationOut="fadeOutUp"
>
  <View style={styles.container}>
    <TouchableOpacity style={styles.itemContainer}>
      <Text style={styles.label}>A</Text>
    </TouchableOpacity>

    <TouchableOpacity style={styles.itemContainer}>
      <Text style={styles.label}>B</Text>
    </TouchableOpacity>

    <TouchableOpacity style={styles.itemContainer}>
      <Text style={styles.label}>C</Text>
    </TouchableOpacity>

    <TouchableOpacity style={styles.itemContainer}>
      <Text style={styles.label}>D</Text>
    </TouchableOpacity>

    <TouchableOpacity
      style={[
        styles.itemContainer,
        { borderBottomLeftRadius: 16, borderBottomRightRadius: 16 },
      ]}
    >
      <Text style={styles.label}>E</Text>
    </TouchableOpacity>
  </View>
</Modal>
```

My custom background looks like this:

```tsx
<View
  style={{
    marginTop: hp("50%"),
    flex: 1,
  }}
>
  <TouchableWithoutFeedback onPress={() => setShowModal(false)}>
    <View
      style={{ backgroundColor: "#000", height: hp("50%"), width: wp("100%") }}
    />
  </TouchableWithoutFeedback>
</View>
```

I would like to be able to interact with the icons in the header (both are already wrapped in Touchable components so already clickable). However when the Modal is visible I am unable to interact with those. I'd be grateful for a solution that helps me bypass this issue. Thank you for taking the time to read!

**Comment by mmazzarolo** [comment]
👋 to clarify, why are you providing a custom backdrop instead of using the default one?

**Reply by sankomil** [reply]
I wanted the to be no dark backdrop for the header aspect of it so that is why. I want to allow the users to be able to interact with the two icons on the top (list and add) while everything else bellow the modal goes under the typical dark backdrop.

**Reply by ahaseeb001** [reply]
did you find any solution?

**Reply by SeongwoonHong** [reply]
@mmazzarolo I also would like to do this however I'm not able to find a way to do so

**Comment by gerryfletch** [comment]
I'm trying to figure this out too. FWIW, my use case is more like Google Maps, where tapping on some element opens a small modal but you can still drag and tap around on the "backdrop" so to speak.
